usr/src/cmd/fs.d/nfs/mountd/mountd.c

```diff
@@ -362,6 +363,20 @@ convert_int(int *val, char *str)
 	return (0);
 }
 
+static void
+create_svcs(const rpcprog_t prognum, const rpcvers_t versnum, int port) {
+	if (svc_create_port(mnt, prognum, versnum, "datagram_v", htons(port)) == 0) {
+		syslog(LOG_ERR, "couldn't register devices on datagram_v %d\n",
```

Comment:
Is the expectation that we're going to be getting a more detailed reason as to *why* this failed from the svc_create_port() function?